### ISSUE TITLE
News webinterface remains blank after upgrade to owncloud 8.0.x

### ISSUE THREAD

**Issue by gjanssens** [owncloud/news#757]
I have recently upgraded from owncloud 7.x to 8.0.0 and shortly after to 8.0.2. Since this upgrade to 8.0.x the owncloud news webinterface stopped working. I can still read the news via the Android Owncloud news app, so it seems the issue is only with the webinterface.

When I select the news app in the webinterface, I get this:
![owncloud news](https://cloud.githubusercontent.com/assets/619393/6764516/3f1b2116-cfb3-11e4-993a-58061afce011.png)

There is no sidebar, and no news messages. I do have several feeds configured (which I can see in the android app), but nothing is visible here. Additonally, in this state I can only click on the cloud image to go back to the default app (Files) or in the search field. Clicking on my user name has no effect (it should display the menu), nor does clicking on the News header (which should pop up a list of available apps).

Information on my installation:
* [owncloud.log](https://cloud.kobaltwit.be:22443/index.php/s/Qcp4d82Dmal3V8J) trimmed so it starts from clicking on the cloud image and then selecting the news app
* Owncloud version: 8.0.2, installed from the isv_ownCloud_desktop yum repository. The issue was already there in owncloud 8.0.0, installed from the same repository (as an upgrade to 7.x).
* News version: 2.002
* Browser and version: Firefox 36 on Fedora linux. I also tried a recent version of google chrome on Windows with the same results.
* php version: php-5.4.15-1.el6.remi.i686
* Distribution: Centos 6 (fully up to date)

**Comment by Raydiation** [comment]
I cant view your log, you should update your cloud ;)

**Comment by Raydiation** [comment]
Also news 2.002 does not work on 8 anymore, upgrade to 5.2.8

**Comment by gjanssens** [comment]
Thanks for the reply. I figured out 2.002 was obsolete myself shortly after opening this issue.

For some reason I thought news was released as part of the owncloud rpm libraries but it appears that was a wrong assumption. So it never occurred to me to upgrade it myself manually.

Anyway, because I'm on Centos 6, I can't upgrade to 5.2.8 due to the stricter libxml dependency. I can upgrade to 3.105 though and that fixes news for me for the time being. I'm adding this info in case others run into this.
